packages/stryker/src/Stryker.ts

```diff
@@ -98,7 +98,11 @@ export default class Stryker {
   }
 
   private removeExcludedMutants(mutants: Mutant[]): Mutant[] {
-    return mutants.filter(mutant => this.config.excludedMutations.indexOf(mutant.mutatorName) === -1);
+    if (typeof this.config.mutator === 'string') {
+      return mutants;
+    }
+    const mutatorDescriptor = this.config.mutator as MutatorDescriptor;
+    return mutants.filter(mutant => mutatorDescriptor.excludedMutations.indexOf(mutant.mutatorName) === -1);
```

Comment:
Please move this block in an "else". That way it is immediately clear that this method has 2 branches and it is not an oversight of the developer.